go.mod

```diff
@@ -102,8 +102,8 @@ require (
 	golang.org/x/sync v0.10.0
 	golang.org/x/sys v0.29.0
 	golang.org/x/time v0.6.0
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20241021214115-324edc3d5d38
-	google.golang.org/grpc v1.69.4
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20241202173237-19429a94021a
```

Comment:
Looks like this dependency was susceptible to GHA script injection (but no mention if it was exploited); https://github.com/googleapis/go-genproto/compare/324edc3d5d38...19429a94021a

- https://github.com/googleapis/go-genproto/pull/1169